packages/frontend/src/features/catalog/components/CatalogTableListItem.tsx

```diff
@@ -53,68 +55,68 @@ export const CatalogTableListItem: FC<React.PropsWithChildren<Props>> = ({
             <Group
                 noWrap
                 position="apart"
-                py="sm"
+                spacing="xs"
                 px="xs"
                 sx={(theme) => ({
-                    minHeight: 40,
-                    borderRadius: theme.radius.sm,
-                    padding: theme.spacing.md,
+                    minHeight: 48,
+                    borderBottom: isLast
+                        ? 'none'
+                        : `1px solid ${theme.colors.gray[2]}`,
                     backgroundColor: hovered
-                        ? theme.colors.gray[2]
-                        : theme.colors.gray[1],
+                        ? theme.colors.gray[1]
+                        : 'transparent',
                     border: isSelected
                         ? `2px solid ${theme.colors.blue[6]}`
                         : undefined,
                     cursor: 'pointer',
+                    borderTopLeftRadius: isFirst ? theme.radius.lg : 0,
+                    borderTopRightRadius: isFirst ? theme.radius.lg : 0,
+                    borderBottomLeftRadius: isLast ? theme.radius.lg : 0,
+                    borderBottomRightRadius: isLast ? theme.radius.lg : 0,
```

Comment:
There's a minor issue here with the last table radius when there is a child. 

<img width="612" alt="Screenshot 2024-05-30 at 13 16 56" src="https://github.com/lightdash/lightdash/assets/1864179/18229aa0-fbaa-49b9-878d-8f584e90759e">

Reply:
Nice! Added a conditional check
<img width="700" alt="image" src="https://github.com/lightdash/lightdash/assets/7611706/2fc3c307-dbdb-4c36-a56d-cbe3e348e3a8">
https://github.com/lightdash/lightdash/pull/10247/commits/c288de2aaa170f87316508384f88b28e6299db2d